dev.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11-slim-buster
+
+WORKDIR /app
+
+
+RUN apt update && \
+    apt install -y libpq-dev python3-dev gcc make  # Install dependencies needed for psycopg2
+
+COPY orchestrator/requirements.txt /app/requirements.txt
+RUN pip install -r /app/requirements.txt --no-cache-dir
+
+COPY ../omotes-sdk-protocol/python/ /omotes-sdk-protocol/python/
+COPY ../omotes-sdk-python/ /omotes-sdk-python/
+RUN pip install -e /omotes-sdk-python/
```

Comment:
Does this `-e` still have any effect here? As the packages have already been copied (and frozen) into the Docker filesystem. Will package changes still be reflected immediately in the running containers?

Reply:
Good question. I suspect not. If a bind mount is used to overwrite /omotes-sdk-python/, then it could matter.

Reply:
I removed the `-e`